src/modules/article/articleApi.ts

```diff
@@ -12,25 +12,36 @@ import {
   fetchAuthorized,
 } from '../../util/apiHelpers';
 import config from '../../config';
+import { ArticleApiType, ArticleSearchResult } from './articleApiInterfaces';
 
 const articleUrl = apiResourceUrl('/article-api/v2/articles');
 
-export const searchArticles = (id, locale, queryString = '') =>
+export const searchArticles = (
+  locale: string,
+  queryString = '',
+): Promise<ArticleSearchResult> =>
   fetchAuthorized(
-    `${articleUrl}/${id}?language=${locale}&fallback=true${queryString}`,
+    `${articleUrl}/?language=${locale}&fallback=true${queryString}`,
```

Comment:
Rart at id var med her. Du er sikker på at den ikkje brukes?

Reply:
Yep, den var ikke i bruk så vidt jeg klarte å søke meg frem til.
Vi har en annen funksjon for å hente med id'er.